pfcpiface/upf.go

```diff
@@ -129,6 +130,8 @@ func NewUPF(conf *Conf, fp datapath) *upf {
 		maxReqRetries:     conf.MaxReqRetries,
 		enableHBTimer:     conf.EnableHBTimer,
 		readTimeout:       time.Second * time.Duration(conf.ReadTimeout),
+		fteidGenerator:    NewFTEIDGenerator(),
+
```

Comment:
The Ling GHA is failing because if this empty line. The following change will fix the issue
```suggestion
```

Reply:
Thanks, fixed